bld/CLMBuildNamelist.pm

```diff
@@ -3014,8 +3015,8 @@ sub setup_logic_do_harvest {
          $cannot_be_true = "$var can only be set to true when running a transient case (flanduse_timeseries non-blank)";
       }
 
-      elsif (!&value_is_true($nl->get_value('use_cn')) && !&value_is_true($nl->get_value('use_fates'))) {
```

Comment:
For the `fates_harvest_mode` update it was decided that we should segregate fates from `do_harvest` to avoid increasing complexity for this update since the `landuse_timeseries` option for fates harvest will be deprecated at a future date.  As such I wanted to avoid allowing `do_harvest` with fates entirely.